src/AltinnCore/Common/Services/Implementation/AuthorizationStudioSI.cs

```diff
@@ -13,7 +13,7 @@
 namespace AltinnCore.Common.Services.Implementation
 {
     /// <summary>
-    /// Authorization service created for service development where test data is located on disk
+    /// Implementation for Authorization created for app development where test data is located on disk
```

Comment:
I would like a uniform comment for all SI-classes. E.g. Studio/App implementation of the authroization service. + potential other useful information in a following sentence.